src/stratis_cli/_actions/_debug.py

```diff
@@ -40,6 +42,25 @@ def refresh_state(_namespace):
         if return_code != StratisdErrors.OK:  # pragma: no cover
             raise StratisCliEngineError(return_code, message)
 
+    @staticmethod
+    def send_uevent(_namespace):
+        """
+        Issue a synthetic uevent from the CLI.
+        """
+        try:  # pragma: no cover
```

Comment:
We do want to throw in one test. It should expect a failure with the exception type ```StratisCliSynthUeventError```. You'll want to add it in in stratis-cli/tests/whitebox/integration/test_debug.py. It's probably correct to make a new class which is a subclass of SimTestCase like the other class. Use examples that you'll find in lots of places that use ```self.check_error```; that's what you should be using here, because you will expect the error. Then try to move the ```no cover``` pragmas around so they allow only the minimal lines of code that will still allow 100% coverage.

Reply:
If I remove the `no cover` on both the "try" and "except" blocks, I get a coverage error on lines 52-58 (which would seem to be most, but not all, of the "try" block).